README.md

```diff
@@ -1 +1,180 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live demo is available [here](https://pierrebarba.github.io/capstone-metrics/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+To clone the repository in local environment try following steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ node --version && npm --version
+```
+### Setup
+
+Clone the repository using [this link](https://github.com/PierreBarba/capstone-metrics)
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/PierreBarba/capstone-metrics.git
+```
+### Get into development
+
+In the terminal, run these commands to get into development.
+```
+$ cd bookstore-with-react
+
+$ npm init -y
+
+$ npm install (install dependencies)
+
+$ npm run build (create the optimized for production)
+
+$ npm start (run the app locally)
+```
+### Usage
+
+This website is applicable for both mobile and desktop version.
+
+### Run tests
+
+### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Pierre Barba**
+
+- GitHub: [@PierreBarba](https://github.com/PierreBarba)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Styling of the webpage**
+- [ ] **Animation to add functionality**
+- [ ] **Adding responsiveness**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+
+Feel free to check the [issues page](https://github.com/PrangonGhose/space-traveller-s-hub/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Authors would like to thank:
+- [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)
+Thank you very much for allowing me to use your design which I took as a reference to carry out my project- Code 
+- Reviewers
+- Coding Partners
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./license) licensed.
```

Comment:
- [ ] When I click the link to the `MIT.md` file that you added, I get this 404 `error`, Please update your Readme, it will be nice to update this section(`License`) with the type of License and a link to the license file, as this will make your readme more informative and professional. 👍🏻